testbed/correctness/correctness_test.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package correctness
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.opentelemetry.io/collector/service/defaultcomponents"
+	"go.opentelemetry.io/collector/testbed/testbed"
+)
+
+type PipelineDef struct {
+	receiver     string
+	exporter     string
+	testName     string
+	dataSender   testbed.DataSender
+	dataReceiver testbed.DataReceiver
+	resourceSpec testbed.ResourceSpec
+}
+
+var correctnessResults testbed.TestResultsSummary = &testbed.CorrectnessResults{}
+
+func TestMain(m *testing.M) {
+	testbed.DoTestMain(m, correctnessResults)
+}
+
+func TestDefaultTracingComponentCombos(t *testing.T) {
```

Comment:
Can we make the test name shorter or give more room for the "Test" field in the test results so that they are nicely aligned? We get this currently:
```
Test                                    |Result|Duration|Sent Items|Received Items|Failure Count|Failures
 
----------------------------------------|------|-------:|---------:|-------------:|------------:|--------
 
DefaultTracingComponentCombos/otlp-jaeger|PASS  |      1s|      4059|          4059|         3247|TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; Kind,1!=0,count=296; Status,(*v1.Status)(nil)!=&v1.Status{Code:0, Message:""},count=232; 
DefaultTracingComponentCombos/zipkin-opencensus|PASS  |      1s|      4059|          4059|        17397|StartTimeUnixNano,0x16195d8be19b922f!=0x16195d8be19b9098,count=8114; Attributes,0!=2,count=3586; Links,2!=0,count=2441; Status,&v1.Status{Code:0, Message:""}!=(*v1.Status)(nil),count=241; TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; Kind,1!=0,count=296; 
DefaultTracingComponentCombos/otlp-opencensus|PASS  |      1s|      4059|          4059|          296|Kind,1!=0,count=296; 
DefaultTracingComponentCombos/jaeger-opencensus|PASS  |      1s|      4059|          4059|         3247|Status,(*v1.Status)(nil)!=&v1.Status{Code:0, Message:""},count=232; TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; Kind,1!=0,count=296; 
DefaultTracingComponentCombos/opencensus-jaeger|PASS  |      1s|      4059|          4059|         3247|Status,(*v1.Status)(nil)!=&v1.Status{Code:0, Message:""},count=232; TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; Kind,1!=0,count=296; 
DefaultTracingComponentCombos/zipkin-otlp|PASS  |      1s|      4059|          4059|        17389|TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; StartTimeUnixNano,0x16195d8cd67f186c!=0x16195d8cd67f17d8,count=8106; Attributes,7!=9,count=3586; Links,1!=0,count=2441; Status,&v1.Status{Code:0, Message:""}!=(*v1.Status)(nil),count=241; Kind,1!=0,count=296; 
DefaultTracingComponentCombos/jaeger-jaeger|PASS  |      1s|      4059|          4059|         3247|TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; Kind,1!=0,count=296; Status,(*v1.Status)(nil)!=&v1.Status{Code:0, Message:""},count=232; 
DefaultTracingComponentCombos/opencensus-opencensus|PASS  |      1s|      4059|          4059|          296|Kind,1!=0,count=296; 
DefaultTracingComponentCombos/otlp-zipkin|PASS  |      1s|      4059|          4059|        17742|TraceState,"lasterror=f39cd56cc44274fd5abd07ef1164246d10ce2955"!="",count=2719; StartTimeUnixNano,0x16195d8d8cf5fe9e!=0x16195d8d8cf60008,count=8110; Attributes,7!=9,count=3935; Links,1!=0,count=2441; Status,&v1.Status{Code:0, Message:""}!=(*v1.Status)(nil),count=241; Kind,1!=0,count=296; 
```

Reply:
Shortened